pkg/documents/documents.go

```diff
@@ -27,20 +27,32 @@ import (
 	"github.com/go-kit/kit/log"
 	"github.com/gorilla/mux"
 	"gocloud.dev/blob"
+	"gocloud.dev/secrets"
 )
 
 var (
 	errNoDocumentID = errors.New("no Document ID found")
 )
 
+type sizeLimit uint64
+
+func (l sizeLimit) Limit() int64 {
+	return int64(l)
+}
```

Comment:
I recommend removing this method and calling `int64(sizeLimit)` directly in the code since it makes it clear that all it's doing is casting.

Reply:
Good call. I'll make the change.